### PR TITLE
Alternative sign-in page using Identity Platform

### PR DESCRIPTION
**Pull request by whaught**
Issue https://github.com/google/exposure-notifications-verification-server/issues/141

## Proposed Changes

* Create an alternative login page
    * Will Unblock us from using the Identity Platform login w/ multi-factor
* At the moment it's overly verbose with an alert bar at the top. We'll make it pretty later.
* Will someday replace our index.html login

Note: FirebaseUI does not support multi-factor auth. That's a shame because it does all the html for us and it is pretty.

Screenshot from 2020-08-18 14-00-35

**Release Note**

```release-note
Create a new login page without firebaseUI
```

### CODE REVIEW

**Comment by icco** [comment]
Is this the same as IAP? I was just reading https://cloud.google.com/load-balancing/docs/negs/serverless-neg-concepts#limitations_with which says cloud run doesn't support IAP.

**Comment by whaught** [comment]
> Is this the same as IAP? I was just reading https://cloud.google.com/load-balancing/docs/negs/serverless-neg-concepts#limitations_with which says cloud run doesn't support IAP.

This is: https://cloud.google.com/identity-platform. I believe IAP uses/depends on this for external identity providers (we're just using email sign-in for now), but we're not using the proxy part

File: cmd/server/assets/login.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+        <div class="card mb-3">
+            <div class="card-body">
+                <form>
+                    {{ .csrfField }}
+                    <div class="row form-group">
+                        <label for="email" class="col-sm-3">Email</label>
+                        <div class="col-sm-9">
+                            <div class="input-group p-0 shadow-sm">
+                                <input type="text" id="email" name="email" class="form-control" />
```
Comment (sethvargo) [inline cmd/server/assets/login.html:26]: Nit: `type=email` is more accurate and will show the right keyboard on mobile

Comment (whaught) [reply cmd/server/assets/login.html:26]: Thanks! I'll incorporate into my next

**Pull request merged**